src/docs/self-hosted/releases.mdx

```diff
@@ -4,14 +4,18 @@ title: 'Self-Hosted Versions & Releases'
 
 ![CalVer: YY.MM.MICRO](https://img.shields.io/badge/calver-YY.MM.MICRO-ad6caa.svg)
 
-Self-hosted Sentry follows a monthly release schedule using the [CalVer](https://calver.org/#scheme) versioning scheme. A new version is released on the [15th of each month](https://github.com/getsentry/onpremise/blob/704e4c3b5b7360080f79bcfbe26583e5a95ae675/.github/workflows/release.yml#L20-L24), and patch releases are done when needed in-between. You can find the [latest release](https://github.com/getsentry/onpremise/releases/latest) over at the [releases section of our onpremise repository](https://github.com/getsentry/onpremise/releases/).
```

Comment:
Removing "patch" here since that implies "stability" more strongly than we'd like.

Reply:
How about "follow-up releases"?

Reply:
Wfm, done in b3969a7d63482da5e8aeefedbdaa6495c22f1fb0.